CSS/styles.css

```diff
@@ -0,0 +1,93 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+html {
+  display: flex;
+  justify-content: center;
+}
+
+body {
+  background-color: #f7f7f9;
+  max-width: 100%;
+  width: 400px;
+}
+
+/* Navbar */
+#header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.5rem;
+  background-color: white;
+}
+
+.header {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.menu {
+  width: 24px;
+  height: 24px;
+}
+
+/* Section */
+.hero-section {
+  background-image: url(../images/header-shapes-mobile.svg);
```

Comment:
[Optional]
- [ ] Please consider fixing the size of the `background-image`, we should try to make the website identical to Figma design 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/191595326-4c248857-d3d9-44b4-873c-5e39a85a2102.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/191595450-50254810-a5a3-4f5a-8209-f0504010e32c.png)